privacy/enforcer.go

```diff
@@ -124,36 +124,35 @@ func conditionToRuleComponentNames(conditions []string) ([]ScopedName, error) {
 	return sn, nil
 }
 
-func activityDefaultToDefaultResult(activityDefault *bool) ActivityResult {
+func activityDefaultToDefaultResult(activityDefault *bool) bool {
 	if activityDefault == nil {
 		// if default is unspecified, the hardcoded default-default is true.
-		return ActivityAllow
-	} else if *activityDefault {
-		return ActivityAllow
+		return true
+	} else {
+		return *activityDefault
 	}
-	return ActivityDeny
 }
 
-func (e ActivityControl) Evaluate(activity Activity, target ScopedName) ActivityResult {
+func (e ActivityControl) Evaluate(activity Activity, target ScopedName) bool {
 	plan, planDefined := e.plans[activity]
 
 	if !planDefined {
-		return ActivityAbstain
+		return true
```

Comment:
This `true` value is the same as `activityDefaultToDefaultResult`. Do you think it might be good to extract that to a constant value? in the future, we may need to replace that constant with a function if some of the activities change to a default of false.